frontend/src/app/data-entry/agent-form/agent-description-form/agent-description-form.component.html

```diff
@@ -0,0 +1,98 @@
+<form>
+    <h3>Designators</h3>
+
+    <p>What expressions are used in the text to refer to this agent?</p>
+
+    <lc-designators-control [formControl]="form.controls.designators"></lc-designators-control>
+
+    <h3>Gender</h3>
+
+    <p>Does the text establish the gender of this agent?</p>
+
+    <div class="mb-4">
+        <label class="form-label" for="input-gender">Gender</label>
+        <select class="form-select" id="input-gender"
+            [formControl]="form.controls.gender.controls.gender">
+            <option *ngFor="let option of genderOptions" [value]="option.value">
+                {{option.label}}
+            </option>
+        </select>
+    </div>
+
+    <div class="mb-4">
+        <label class="form-label" for="input-gender-mention">
+            How is this information presented in the text?
+        </label>
+        <select class="form-select" id="input-gender-mention"
+            [formControl]="form.controls.gender.controls.sourceMention">
+            <option *ngFor="let option of genderSourceMentionOptions" [value]="option.value">
+                {{option.label}}
+            </option>
+        </select>
+    </div>
+
+    <div class="mb-4">
+        <label class="form-label" for="input-gender-notes">
+            Additional notes
+        </label>
+        <textarea class="form-control" id="input-gender-notes"
+            [formControl]="form.controls.gender.controls.note">
+        </textarea>
+    </div>
+
+    <ng-container *ngIf="isGroup$ | async">
```

Comment:
Currently, the location form does not appear when I select `isGroup`. It only determines whether to hide/show the form on initial load.

Reply:
True, there is a similar issue where the agent's name in the title / breadcrumb does not update when you change it in the form.

The reason is that the data is fetched from the API but the form isn't sending data yet. I think the best way to resolve this is by controlling when Apollo refreshes query results, right?